Talvihoitoreitit - lataa kalustomäärä exceliin

rckopteri opened the pull request:
Exceliin ladattaessa kalustojen määrät olivat toteutettu vielä vanhalla tietorakenteella, eikä niiden oikea määrä tullut mukaan exceliin

rckopteri committed:
Exceliin ladattaessa kalustojen määrät olivat toteutettu vielä vanhalla tietorakenteella, eikä niiden oikea määrä tullut mukaan exceliin

jarnovayrynen commented on src/clj/harja/palvelin/palvelut/laadunseuranta/talvihoitoreitit_excel.clj:
```diff
@@ -141,83 +141,73 @@
-                                                  (:hoitoluokka rivi)
-                                                  (:pituus rivi)]})
-                                    (:reitit reitti))))
+    #_(oikeudet/vaadi-kirjoitusoikeus oikeudet/urakat-laadunseuranta-talvihoitoreititys user urakka-id)
```
Onko oikeustarkistus tarkoituksella kommentoituna pois?

rckopteri replied:
Kyllä. On ollut puhetta, että jossain vaiheessa ominaisuus otettaisiin laajemminkin käyttöön. Esim urakoitsijat päästettäisiin näkemään, että mitä rajapinnat tuottaa. Mutta ei vielä

jarnovayrynen commented on src/clj/harja/palvelin/palvelut/laadunseuranta/talvihoitoreitit_excel.clj:
```diff
@@ -141,83 +141,73 @@
+                                 :raportin-yleiset-tiedot {:raportin-nimi (str (:nimi urakan-tiedot) "- Talvihoitoreitit")
+                                                           :urakka (:nimi urakan-tiedot)}}]
+                     (if (empty? taulukot)
+                       [[:taulukko {} nil [["Ei kustannuksia valitulla aikavälillä"]]]]
```
Onko tämä termi "ei kustannuksia" oikein? Kun taulukoiden sisältö vaikuttaa olevan kalusto-taulukko ja reitti-taulukko.. toisaalta mietin, voisiko tuo mennä :taulukon optioihin, siellä on :tyhjä optio erikseen olemassa.

rckopteri replied:
Mainiota. Ei tosiaan ole. Pitää puhua talvihoitoreiteistä.

rckopteri committed:
Tarkenna käyttöoikeuksien tarkistusta

rckopteri committed:
Lisää exceliin parempi teksti, jota käyttäjät eivät koskaa näe.

Exceliä ei voi siis tulostaa, jos reittejä ei ole, mutta parmepi, että korjataan teksti silti.
